src/custom-elements/src/components/carousel/carousel.tsx

```diff
@@ -44,6 +44,10 @@ export class Carousel {
    * To render Carousel with pagination indicators
    */
   @Prop() pagination: boolean;
+  /**
+   * To render Carousel without Button Controllers
+   */
+  @Prop() hideButtonControllers = false;
```

Comment:
@tomasjimenez I would change camel case to kebab-case for consistency wth rest of Chi wc